android_client/app/build.gradle

```diff
@@ -23,12 +32,12 @@ android {
 
     buildTypes {
         debug {
-            signingConfig signingConfigs.debug
+            signingConfig signingConfigs.debugSignConfig
             minifyEnabled false
         }
 
         release {
-            signingConfig signingConfigs.debug
+            signingConfig signingConfigs.debugSignConfig
```

Comment:
should we add a release sign config ?

Reply:
I don't think we need to worry about this now, until we decide to turn this app source code to internal.